### PR TITLE
Update auth and make-token usage

### PR DESCRIPTION
**Pull request by bburtin**
Update command line usage for the auth and make-token commands.
Describe the difference between API tokens and launch tokens.  Document
the BRKT_API_TOKEN environment variable.

### CODE REVIEW

File: brkt_cli/auth.py [hunk 1]
```diff
@@ -42,8 +42,11 @@ def register(self, subparsers, parsed_config):
             self.name(),
             description=(
                 'Authenticate with the Bracket service. On success, print '
-                'the API token (JSON Web Token) that can be used to make '
-                'calls to Bracket REST API endpoints.'
+                'the API token (JSON Web Token) that is used for making '
+                'calls to Bracket REST API endpoints. Set the '
```
Comment (bonzotronix) [inline brkt_cli/auth.py]: s/Set/You must set/  (and could omit the comma after 'value' below)

File: brkt_cli/make_token.py [hunk 2]
```diff
@@ -126,10 +126,14 @@ def _setup_args(subparsers):
     parser = subparsers.add_parser(
         SUBCOMMAND_NAME,
         description=(
-            'Generate a launch token (JSON Web Token) for encrypting an '
-            'instance or launching an encrypted instance. If a signing key is '
-            'not specified, get a token from the Bracket service. '
-            'A timestamp can be either a '
+            'Generate a launch token (JSON Web Token), '
+            'used by the Metavisor to communicate with the Bracket Service. '
```
Comment (bonzotronix) [inline brkt_cli/make_token.py]: Nit:  looks like we lower-case 'service' elsewhere.
(I guess we're not saying "MCP" anymore?)

File: brkt_cli/make_token.py [hunk 3]
```diff
@@ -126,10 +126,14 @@ def _setup_args(subparsers):
...
+            'The encrypt and update commands generate a launch token '
+            'automatically.  Users who need fine-grained control of '
+            'their launch tokens can optionally use this command and '
+            'pass the generated launch token during encryption or update. If '
```
Comment (bonzotronix) [inline brkt_cli/make_token.py]: Is it worth squeezing in another sentence here about using the token with `make-user-data`?
"The launch token can also be used as input to the make-user-data command."

File: brkt_cli/auth.py (hunk 1 above)
Comment (bburtin) [inline brkt_cli/auth.py]: Done.

File: brkt_cli/make_token.py (hunk 2 above)
Comment (bburtin) [inline brkt_cli/make_token.py]: Done.

**Comment by bonzotronix** [comment]
FTP

File: brkt_cli/make_token.py (hunk 3 above)
Comment (bburtin) [inline brkt_cli/make_token.py]: Done.

**Pull request merged**